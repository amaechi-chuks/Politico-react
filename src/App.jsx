import React, { Component } from 'react';
import { Route, Switch, Redirect } from 'react-router-dom';
import { Provider } from 'react-redux';
import Notifications from 'react-notify-toast';
import { HomePage } from './HomePage';
import Login from './LoginPage/LoginPage';
import Layout from './Containers/Home/Layout';
import Signup from './Signup/Signup';
import NotFound from './PageNotFound/PageNotFound';
import UserProfile from './Userprofile/UserProfile';
import store from './_helpers/store';

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <React.Fragment>
        <Provider store={store}>
          <Notifications />
          <Switch>
            <Route exact path="/" component={HomePage} />
            <Route path="/login" component={Login} />
            <Route path="/signup" component={Signup} />
            <Route
              path="/user"
              component={UserProfile}
              // render={() => <Layout />}
            />
            <Route path="/notFound" component={NotFound} />
            <Redirect to="notFound" />
          </Switch>
        </Provider>
      </React.Fragment>
    );
  }
}
export default App;
