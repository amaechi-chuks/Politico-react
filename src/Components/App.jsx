/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
// import Header from './Header/Header';

export default class App extends Component {
  render() {
    return (
      <div>
      <h1>hello</h1>
      </div>
    )
  }
}

