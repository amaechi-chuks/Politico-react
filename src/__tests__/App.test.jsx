import React from 'react';
import { shallow } from '../enzyme';

import App from '../components/App';

describe('List tests', () => {
  it('renders without crashing', ()=> {
    shallow(<App />);
  })
});
