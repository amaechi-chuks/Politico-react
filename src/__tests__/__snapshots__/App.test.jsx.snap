// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match snapshot 1`] = `ShallowWrapper {}`;

exports[`renders without crashing 1`] = `ShallowWrapper {}`;
